scene/gui/color_picker.cpp

```diff
@@ -1414,6 +1531,81 @@ void ColorPicker::_uv_input(const Ref<InputEvent> &p_event, Control *c) {
 			emit_signal(SNAME("color_changed"), color);
 		}
 	}
+
+	// TODO: Is it better to cast and check it or to not check it and get vector?
```

Comment:
You need to check whether the arriving even is a pressed action. But afterwards you could probably use `Input.get_vector()` and it might yield better results.